### PR TITLE
51570 remove v0 eligibility call for v2

### PR DESCRIPTION
**Pull request by simiadebowale**
This PR removes the errant v0 cc eligibility call we were seeing in v2 and replaces that call with the required v2 cc eligibility check.

### CODE REVIEW

File: src/applications/vaos/new-appointment/redux/actions.js [hunk 1]
```diff
@@ -717,11 +720,15 @@ export function checkCommunityCareEligibility() {
...
+        let response = null;
+        if (useV2) {
+          response = await getCommunityCareV2(getCCEType(state));
+        } else {
+          response = await getCommunityCare(getCCEType(state));
+        }
         dispatch({
           type: FORM_UPDATE_CC_ELIGIBILITY,
-          isEligible: response.eligible,
+          isEligible: response.eligible || response.length > 0,
```
Comment (jenniemc) [inline src/applications/vaos/new-appointment/redux/actions.js:731]: If latest v2 cc eligibility returns a boolean, why do we need a check for array length?

**Review by jenniemc — COMMENTED** [review]
I just have one question.

File: src/applications/vaos/new-appointment/redux/actions.js (hunk 1 above)
Comment (simiadebowale) [reply src/applications/vaos/new-appointment/redux/actions.js:731]: Good catch @jenniemc -  I meant to take that out

**Review by jenniemc — APPROVED** [review]
LGTM!

**Pull request merged**